hack/update-codegen.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
```

Comment:
@dvaldivia I guess this is from another place? Do we want to mix the licenses or rewrite something on our own?

Reply:
I guess this file was not expected to be checked in.

Reply:
updated